Sources/SmileID/Classes/Metadata/Metadatum.swift

```diff
@@ -3,4 +3,15 @@ import Foundation
 public struct Metadatum: Codable {
     let name: String
     let value: CodableValue
+    let timestamp: String
+
+    init(
+        key: MetadataKey,
+        value: CodableValue,
+        date: Date
+    ) {
+        self.name = key.rawValue
+        self.value = value
+        self.timestamp = date.toISO8601WithMilliseconds(timezone: .current)
```

Comment:
```suggestion
        self.timestamp = date
```
if we set the correct timestamps initially we don't need to do it here.